charts/drax/charts/dashboard/templates/configmap-server.yaml

```
@@ -11,38 +11,48 @@ top:
 
 name: {{ include "accelleran.common.fullname" (dict "top" $) }}-server
 
+{{- $config := .Values.config }}
+
+# Define required variables
+{{- $requiredVariables := list
+  "defaultOranNamespace"
+  "kafkaURL"
+  "kafkaPort"
+  "svcOrchestratorHost"
+  "svcOrchestratorPort"
+  "networkStateMonitorHost"
+  "networkStateMonitorPort"
+  "configApiHost"
+  "configApiPort"
+  "svcMonitorHost"
+  "svcMonitorPort"
+  "pcixAppPodName"
+  "pcixAppPort"
+  "ksqldbPodName"
+  "ksqldbPort"
+  "LteRadioControllerLabel"
+  "draxVersionConfigmap"
+  "grafanaPort"
+}}
+
 data:
   config.json: |
     {
       "production": {
-        "config_id": "{{ .Values.config.config_id }}",
-        "defaultServiceName": "{{ include "accelleran.common.service.name" (dict "top" $) }}",
+        "serviceName": "{{ include "accelleran.common.service.name" (dict "top" $) }}",
+        "servicePort": "{{ .Values.service.ports.http.port }}",
         "defaultServiceNamespace": "{{ tpl .Release.Namespace . }}",
```

Maybe we should add this too to the values file and then with a call `tpl`. Now `tpl` is only overhead as there is nothing to template

---

These variables represent the cluster configuration for the swagger documentation.

As they are automatically generated at deployment time, I don't think we need to expose them for the user as something configurable. (That's why I wanted to keep them internally)

If the previous Is correct to we have a benefit of putting them in the values.yaml?

---

Okay, wasn't fully aware about it's use. Then let's just get rid of the `tpl` call.

```suggestion
        "defaultServiceNamespace": "{{ $.Release.Namespace . }}",
```

---

Not sure if this is really what we want after the above explanation...